test/verify/check-storage-mdraid

```diff
@@ -343,6 +343,44 @@ class TestStorageMdRaid(storagelib.StorageCase):
         # The last disk can not be removed
         b.wait_visible('#detail-sidebar .sidepanel-row:contains(DISK2) button:disabled')
 
+    def testBitmap(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        # Make two huge block devices, so that we can make a array
+        # that is beyond the threshold where Cockpit and mdadm start
+        # to worry about bitmaps.  The backing files are sparse, so
+        # this is okay as long nobody actually writes a lot to these
+        # devices.
+
+        dev1 = self.add_loopback_disk(110000)
+        dev2 = self.add_loopback_disk(110000)
+
+        # We need to use "--assume-clean" so that mdraid doesn't try
+        # to write 110GB to one of the devices during synchronization.
+
+        m.execute(f"mdadm --create raid0 --level=1 --assume-clean --run --raid-devices=2 {dev1} {dev2}")
```

Comment:
This is rather confusing named, it's a `raid1` not raid0. Or did you intend to also add a test for `raid0` not showing a bitmap warning? (Or is it more a naming thing, we start with 0 :) )

Reply:
Ahh, yeah, that got me confused as well just now.  "raid0" is the name, but I first thought it is the type and wondered how this will all work... Yes, let's use a different name.